MtfhReportingDataListener/Domain/TenureChangeEvent.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Hackney.Shared.Tenure.Boundary.Response;
+
+namespace MtfhReportingDataListener.Domain
+{
+    public class TenureChangeEvent
+    {
```

Comment:
Can we not use the [EventData](https://github.com/LBHackney-IT/mtfh-reporting-data-listener/blob/master/MtfhReportingDataListener/Boundary/EntityEventSns.cs) object rather than building a new one?

(I'm suggesting this as we would like to make this as generic as possible)

Reply:
I didn't want to tightly couple the sns message structure to the message we send to kafka.